jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py

```diff
@@ -21,6 +21,11 @@
 CRO_PORTAL = os.environ['CRO_PORTAL']
 CRO_COST_OVER_USAGE = os.environ['CRO_COST_OVER_USAGE']
 CRO_ES_INDEX = os.environ['CRO_ES_INDEX']
+AWS_ACCESS_KEY_ID_ATHIRUMA_BOT = os.environ['AWS_ACCESS_KEY_ID_ATHIRUMA_BOT']
+AWS_SECRET_ACCESS_KEY_ATHIRUMA_BOT = os.environ['AWS_SECRET_ACCESS_KEY_ATHIRUMA_BOT']
```

Comment:
AWS_ACCESS_KEY_ID_ATHIRUMA_BOT, AWS_ACCESS_KEY_ID_ATHIRUMA_BOT
Are those the same as:
ATHENA_ACCOUNT_ACCESS_KEY, ATHENA_ACCOUNT_SECRET_KEY

Reply:
Yes, I am using my creds to querying the Athena database.

Reply:
Why 2 different names ?

Reply:
these are my personal creds, So I named them as my creds.
We need to pass the Athena creds. So I am asking for the Athena creds.